package.json

```diff
@@ -21,9 +21,12 @@
     "lint": "turbo run lint",
     "lint:fix": "yarn lint -- --fix",
     "test": "turbo run test",
+    "test:unit": "turbo run test --filter langchain --",
```

Comment:
This is not quite true, we do have commands to run unit tests and integration tests, but they are inside the `langchain` folder

Reply:
Specifically it's the next line that isn't correct, it can instead be `yarn workspace langchain run test:integration`

Reply:
Yes, I actually went down the path of breaking out linting, unit tests, and integration tests into separate steps, but ultimately didn't think it provided much additional value when I noticed that you had the main parts covered in the CI action.

Reply:
Oh, sorry I re-read that and I think I misunderstood what you were saying. I'll try to clean that up tomorrow. My goal was to have everything run from the root dir by default.